nengo_gui/components/value.py

```diff
@@ -95,6 +95,8 @@ def default_output(obj):
         if isinstance(obj, spa.module.Module):
             if 'default' in obj.outputs.keys():
                 output = obj.outputs['default'][0]
+            elif isinstance(obj, spa.BasalGanglia):
+                output = obj.output
```

Comment:
I think this should be the first thing checked in the if, just in case in the future someone defines a default output for the BasalGanglia and is surprised that the plots suddenly stop working.